### PR TITLE
clean up hero and footer

### PR DESCRIPTION
**Pull request by swapagarwal**
- remove github button from hero
- remove logo from footer
- reorder items in footer

### CODE REVIEW

File: src/pug/includes/body.pug [hunk 1]
```diff
@@ -89,27 +81,21 @@
...
-			img(src="/assets/img/devswag-logo.svg", alt="DevSwag")
 		.item.flex
 			p Made with ❤︎ by #[a(href="https://swapnil.net/") Swapnil Agarwal]
 		.item.flex
-			p Designed by #[a(href="https://github.com/zac-garby") Zac Garby] and #[a(href="https://mihir.ch/") Mihir Chaturvedi]
-		.item.flex
-			p Redesign by #[a(href="http://github.com/anhek") Antonin Nhek]
+			p Designed by #[a(href="https://github.com/zac-garby") Zac Garby], #[a(href="https://mihir.ch/") Mihir Chaturvedi], and #[a(href="http://github.com/anhek") Antonin Nhek]
 		.item.flex
 			p Maintained by #[a(href="https://github.com/aslafy-z") Zadkiel] and #[a(href="https://github.com/vikaspotluri123") Vikas Potluri]
```
Comment (vikaspotluri123) [inline src/pug/includes/body.pug:89]:
```suggestion
			p Maintained by #[a(href="https://github.com/aslafy-z") Zadkiel] and #[a(href="https://vikaspotluri.me") Vikas Potluri]
```

Finally got around to making my website 😄

**Comment by vikaspotluri123** [comment]
I'd like to get #654 merged and test it with this PR if that's okay with you 😇

File: src/pug/includes/body.pug (hunk 1 above)
Comment (swapagarwal) [reply src/pug/includes/body.pug:89]: Nice!

**Comment by swapagarwal** [comment]
@vikaspotluri123 Sure.

**Pull request merged**